config/prow/plugins.yaml

```diff
@@ -1100,6 +1100,13 @@ cherry_pick_unapproved:
 
     For details on the patch release process and schedule, see the [Patch Releases](https://k8s.io/releases/patch-releases) page.
 
+# TODO: enable when https://github.com/kubernetes/test-infra/pull/31286 merged
+# cherry_pick_approved:
+# - org: kubernetes
+#   repo: kubernetes
+#   owners_aliases_url: https://raw.githubusercontent.com/kubernetes/kubernetes/master/OWNERS_ALIASES
```

Comment:
I have two concerns about this:
- This is pulling from GitHub which is prone to errors and flakes (e.g. rate limits, `raw.githubusercontent.com` being unavailable)
- The original idea behind this plugin was to allow more broader group to approve cherry-picks. Concretely, we could allow Release Manager Associates that are good at reviewing cherry-picks to approve them before becoming fully-fledged Release Managers. I think that would be appreciated by both RMs and RMAs: RMs would have help from RMAs, RMAs would have more tasks to work on

That said, it might be better if we have explicit list of folks allowed to approve cherry-picks.

I see two risks with this though:
- We need to keep the list up to date manually, but given that the Release Management team is not changed too often and that we have onboarding/offboarding checklists -- I don't see this being a problem
- We need to make sure that someone who's not part of the Release Management team doesn't add themselves to the list. I think that Prow config changes should be already strict, but this is the risk that we have to keep in mind

Reply:
I don't agree with all of the points, but generally I'd agree that a list might be safer to parse. Switching it over to a manually maintained list.